### PR TITLE
Functional Monitoring documentation update (1 of 5)

### PR DESCRIPTION
**Pull request by chris-campbell9**
Jira:
https://www.mulesoft.org/jira/browse/DOCS-11469

**DO NOT MERGE UNTIL RELEASE!**

### CODE REVIEW

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 1]
```diff
@@ -5,43 +5,57 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can add a schedule to an existing monitor if either of the following two conditions is true:
+You can add a schedule to an existing monitor if either of the following two conditions are true:
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-add-schedule.adoc]: "either...is" (either one is):
```suggestion
You can add a schedule to an existing monitor if either of the following two conditions is true:
```

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 2]
```diff
@@ -5,43 +5,57 @@ endif::[]
...
 
-* You plan to run the test associated with the schedule in a public location and the allocation of schedules for your organization in Anypoint Platform is not exhausted.
+* You plan to run the test associated with the schedule in a public location, and the allocation of schedules for your Anypoint Platform organization has not been exhausted.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-add-schedule.adoc]: Just to stick with plain present tense (as possible or logical)...:
```suggestion
* You plan to run the test associated with the schedule in a public location, and the allocation of schedules for your Anypoint Platform organization is not exhausted.
```

Comment (kbostrom) [reply modules/ROOT/pages/afm-add-schedule.adoc]: This helps a bit with readers for whom English is not a native language. Verb tense is so different among various languages!

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 3]
```diff
@@ -5,43 +5,57 @@ endif::[]
...
-There are two different methods that you can use to add a schedule.
-
-To add a schedule from the *Schedules* section of a monitor:
-
 . Select the monitor in which you want to add a schedule.
-. Click the *Schedules* tab.
-. Click *Add Schedule*.
-. In the Add Schedule dialog, specify this information:
+. Click *Add Schedule* from the *Schedules* table.
+. In the Add Schedule dialog, specify the following:
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-add-schedule.adoc]: If Add Schedule is a visible label, then use bold.

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 4]
```diff
@@ -5,43 +5,57 @@ endif::[]
...
+* The location in which to run the test
+* The interval at which to run the test:
++
+|===
+|Interval |Required Information |Defintion
 
-To add a schedule from the list of schedules that use a particular location:
+|Hourly
+|Minutes
+|The number of minutes after each hour when the test runs, beginning 60 minutes after the schedule has been created
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-add-schedule.adoc]: I have a couple of questions about the meaning and wording of this sentence.
Does "after each hour" refer to "clock hours" 1, 2, 3...or to 60 minutes passing?
Depending on the answer to that, does this mean that the test begins exactly 60 minutes after the schedule is created (what does that mean? What signifies the completion of the creation?) or that 60 minutes after creation, the test period begins but does not initiate until the specified number of minutes after "the hour" (whether that is the clock hour or the 60 minutes)?

Comment (kbostrom) [reply modules/ROOT/pages/afm-add-schedule.adoc]: Maybe we need a SHORT example?

Comment (kbostrom) [reply modules/ROOT/pages/afm-add-schedule.adoc]: In any case, ditto simple present tense:
```suggestion
|The number of minutes, after each hour, that triggers the test, beginning 60 minutes after the schedule is created.
```

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 5]
```diff
@@ -5,43 +5,57 @@ endif::[]
...
+
+|Daily
+|Time
+|The time (hour and minutes) in a day when the test runs
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-add-schedule.adoc]:
```suggestion
|The time each day (hour and minutes) at which the test begins.
```

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 6]
```diff
@@ -5,43 +5,57 @@ endif::[]
...
+
+|Weekly
+|Day of week, Time
+|The day of the week and time when the test runs
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-add-schedule.adoc]:
```suggestion
|The day of the week and the time of that day that initiates the test
```

Comment (kbostrom) [reply modules/ROOT/pages/afm-add-schedule.adoc]: Maybe we could add a short example to each of these definitions?
The example would need to be formatted as if in syntax:

The day of the week and the time of that day that initiates the test, for example `monday`, `12:30`

Comment (kbostrom) [reply modules/ROOT/pages/afm-add-schedule.adoc]: ...and I just thought of something: 12- or 24-hour clock?

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 7]
```diff
@@ -5,43 +5,57 @@ endif::[]
...
+|Hourly
+|Minutes
+|The number of minutes after each hour when the test runs, beginning 60 minutes after the schedule has been created
+
+|Daily
+|Time
+|The time (hour and minutes) in a day when the test runs
+
+|Weekly
+|Day of week, Time
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-add-schedule.adoc]:
```suggestion
|Day of the week and time of day 
```
...or does the table column Required Information take values that appear onscreen or in syntax? In other words, is this capitalized and separated by a comma to indicate two separate values indicates somewhere rather than "general" narrative required info?

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 8]
```diff
@@ -5,43 +5,57 @@ endif::[]
...
+|The day of the week and time when the test runs
+
+|Monthly
+|Day of month, Time
+|The day of the month and time when the test runs
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-add-schedule.adoc]: ditto Daily edits ^

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 9]
```diff
@@ -5,43 +5,57 @@ endif::[]
...
+
+|Advanced
+|Cron
+|The cron expression describing when the test runs
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-add-schedule.adoc]: Alternative:
```suggestion
|The cron expression for the time that initiates the test
```

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 10]
```diff
@@ -5,43 +5,57 @@ endif::[]
...
+
+|===
+
+* Set Starting at (min)
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-add-schedule.adoc]: If any of "Starting at (min)" is a viewable UI label, format in bold.

Comment (kbostrom) [reply modules/ROOT/pages/afm-add-schedule.adoc]: Also, this does not look parallel to its peer list items: it looks like an imperative (step)

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 11]
```diff
@@ -5,43 +5,57 @@ endif::[]
...
+* Set configuration
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-add-schedule.adoc]: ditto

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 12]
```diff
@@ -5,43 +5,57 @@ endif::[]
...
-. In the Add Schedule dialog, specify this information:
-+
-* The name of the monitor to apply the schedule to
-* The test version that you want the schedule to run
-* The frequency at which to run the test
 . Click *Add*.
 
 == Result
-The countdown to the first run of the selected test version begins.
+The countdown to the first run of the selected test begins.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-add-schedule.adoc:53]:
```suggestion
The 60-minute countdown to the first run of the selected test begins.
```

**Review by kbostrom — APPROVED** [review]
CE, CU add schedule .adoc

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 13]
```diff
@@ -3,35 +3,31 @@ ifndef::env-site,env-github[]
...
 
 == Before You Begin
 
-* Ensure that your user ID has the Anypoint Monitoring User permission and is assigned the Exchange Administrators role.
-+
-An administrator for your organization can grant you this permission in Anypoint Access Management.
-* Determine which endpoints you want to test and the REST methods that you want to test them with.
-* Decide upon the return codes that you expect the methods to return.
-* Figure out the frequency at which you would like the monitor to test the endpoints.
+* Ensure that your user ID has the Anypoint Monitoring User permission and that it has been assigned the Exchange Administrators role. An administrator for your organization can grant this permission in Anypoint Access Management. For details, see xref:access-management::roles.adoc[Roles].
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc]: simple present tense
```suggestion
* Ensure that your user ID has the Anypoint Monitoring User permission and that it is assigned the Exchange Administrators role. An administrator for your organization can grant this permission in Anypoint Access Management. For details, see xref:access-management::roles.adoc[Roles].
```

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 14]
```diff
@@ -3,35 +3,31 @@ ifndef::env-site,env-github[]
...
+* Determine which endpoints to test and the REST methods that test them.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc]:
```suggestion
* Determine which endpoints to test and by which REST methods to test them.
```

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 15]
```diff
@@ -3,35 +3,31 @@ ifndef::env-site,env-github[]
...
+* Determine the expected return status codes.
+* Determine the test interval.
 +
-The shortest interval is every fifteen minutes. You can also choose to run the monitor hourly, daily, weekly, or monthly. The schedule begins immediately after you create the monitor. You can schedule another monitor later for the same test suite that is more detailed, if you prefer. You can also modify the schedule after you create the monitor, if you want to specify any of these options:
+The shortest interval is fifteen minutes. Monitors can also run hourly, daily, weekly, or monthly. The schedule begins immediately after creating the monitor, and you can modify the schedule after the monitor has been created. For example:
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc]: In general, we spell out single-digit amounts but use digits in mixed single-digit and multidigit contexts.
We use numerals for all _values_ (as opposed to _amounts_: a subtle distinction sometimes).

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 16]
```diff
@@ -3,35 +3,31 @@ ifndef::env-site,env-github[]
...
+
 ** Set an hourly schedule to start at a certain number of minutes after the hour begins.
 ** Set a daily, weekly, or monthly schedule to start at a certain time of day.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc:20]: These seem clear to me. Maybe use them throughout?

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 17]
```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
-== Result
-
-The dashboard opens. The monitor that you created is listed on the left.
-
-Beneath the name of the monitor is an indicator that explains when the current run of the monitor began. A line at the bottom of the list item shows how much time remains before the next run of the monitor. Also, the *Monitored Endpoints*, *Tests*, and *Schedules* sections will all be empty.
-
-After the monitor has run, the list item for the test suite will indicate whether the monitored endpoints passed or failed.
+. Click *Create Monitor* in the *Functional Monitoring* section of Anypoint Monitoring.
+. Under *Step 1 - Setup the monitor*, specify a monitor name. Additionally, toggle *Validate SSL Certificate* to specify if the monitor needs to use self-signed certifications.
+. Tap the *Step 2 - Select endpoints* tab.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc]: We typically use "click" instead of "tap."

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 18]
```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
+. Populate the fields with the information gathered in "Before you Begin".
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc]: Periods and commas go inside quotation marks, unless what is inside those marks can be mistaken for including the comma (like syntax or similar).
```suggestion
. Populate the fields with the information gathered in "Before You Begin."
```

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 19]
```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
++
+You can preview the test by tapping *Run now* under *Preview the result of your monitor*.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc]: GLOBALLY search for "tap" and replace.
Up to this point, we have limited our UI verbs to "click" to initiate, "select" for menu or list choices, "check" and "uncheck" for checkboxes, and "toggle" for toggles.
Isn't "tap" a mobile-app verb?

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 20]
```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
-[IMPORTANT]
-====
-Tests silently fail when they take more than 120 seconds to run. As a workaround, you can split a long-running test into multiple tests. Download the monitor, split the test across multiple monitors in the BAT CLI, run and time each test to be sure that it is under 120 seconds, and then upload the monitors.
+. Tap the *Step 3 - Set notifications* tab.
+. Optionally configure a notification.
+. Click *Create Monitor*.
 
-If you do not want to split a long-running test, download the monitor and run it from the BAT CLI.
-====
+== Result
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc]: Ditto what I said in a previous edit about this as a subhead.

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 21]
```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
 
-== What to do next
+The *Monitor Detail* view includes the following actions:
 
-Click the *Scheduling* tab to see the schedule for the monitor. You can add additional schedules and delete schedules that you no longer want to run. You can also disable schedules, if you want to pause them, and then enable them again.
+* Tap *Run* (image:afm-ui-run-button.png[width=1%,height=1%]) to immediately execute a monitor.
+* Tap *View Monitor History* (image:afm-ui-history-button.png[width=1.3%,height=1.3%]) to view a list of recent monitor runs.
+* Tap *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) followed by *Edit Monitor* to modify a monitor.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc]: In addition to using "click" instead, consider ">" instead of "followed by":
```suggestion
* Click *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) > *Edit Monitor* to modify a monitor.
```

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 22]
```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
+* Tap *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) followed by *Download Monitor* to download a monitor as BAT source.
+* Tap *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) followed by *Delete Monitor* to delete a monitor.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc]: Ditto.

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 23]
```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
 
-After the monitor runs, click the *Monitored Endpoints* tab to see:
+Additional displayed information includes:
 
-* Whether all endpoints returned the expected codes, as indicated by *PASSED* in the first column of the table
-* Statistics about the execution of the test of the endpoint
+* The most recent test duration
+* Time elapsed since the monitor started running
+* The average response time for each activity (for example, DNS lookup), separated by day
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc:71]: Maybe "per day"? "separated by day" might be confused by nonnative readers or someone not familiar with the common phrase "separated by commas."

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 24]
```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
+* The previous five test execution results, as indicated by *PASSED* or *FAILED* (depending on if the endpoints returned expected codes)
+* A table with the current test schedules
+* Each endpoint asserted with its HTTP method
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc:74]: Is "asserted" the correct jargon? Just checking....

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 25]
```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
-After the monitor runs, click the *Monitored Endpoints* tab to see:
+Additional displayed information includes:
 
-* Whether all endpoints returned the expected codes, as indicated by *PASSED* in the first column of the table
-* Statistics about the execution of the test of the endpoint
+* The most recent test duration
+* Time elapsed since the monitor started running
+* The average response time for each activity (for example, DNS lookup), separated by day
+* The previous five test execution results, as indicated by *PASSED* or *FAILED* (depending on if the endpoints returned expected codes)
+* A table with the current test schedules
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc]: "with" > "displaying"?

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 26]
```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
+* Each endpoint asserted with its HTTP method
 
-As your monitor continues to run at the scheduled intervals, the statistics in this table are updated. They show the minimum, average, and maximum runtime for up to five executions of the test.
+Return to the homepage by tapping *Back to Monitors*.
 
-Click *Tests* to see statistics and other information about the monitor. Because you have only one test in the monitor that you just created, the statistics and information in this section apply to the test, too.
+[NOTE]
+====
+Tests silently fail when run time exceeds 120 seconds. As a workaround, download the monitor and split the tests across multiple monitors via the BAT CLI. Run and time each test to ensure their running times are under 120 seconds, then upload the monitors.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc]: Avoid using ", then" alone as a conjunction:
```suggestion
Tests silently fail when run time exceeds 120 seconds. As a workaround, download the monitor and split the tests across multiple monitors via the BAT CLI. Run and time each test to ensure their running times are under 120 seconds, and then upload the monitors.
```
OR
```suggestion
Tests silently fail when run time exceeds 120 seconds. As a workaround, download the monitor and split the tests across multiple monitors via the BAT CLI. Run and time each test to ensure their running times are under 120 seconds; then, upload the monitors.
```

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 27]
```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
 
-If you want to delete the monitor, delete all versions of it in the *Tests* section.
+Alternatively, to run a monitor more than 120 seconds download the monitor and run it from the BAT CLI.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-monitor.adoc]:
```suggestion
Alternatively, to run a monitor for more than 120 seconds, download the monitor and run it from the BAT CLI.
```

**Review by kbostrom — DISMISSED** [review]
CE

File: modules/ROOT/pages/afm-create-monitor.adoc (hunk 24 above)
Comment (chris-campbell9) [reply modules/ROOT/pages/afm-create-monitor.adoc:74]: Hey, yup it's a concept that is used throughout the UI.

File: modules/ROOT/pages/afm-add-schedule.adoc (hunk 12 above)
Comment (chris-campbell9) [reply modules/ROOT/pages/afm-add-schedule.adoc:53]: The 60-minute countdown only applies if the user selected the Hourly monitor schedule, and this info is contained in the Hourly interval table row.

File: modules/ROOT/pages/afm-create-monitor.adoc (hunk 23 above)
Comment (chris-campbell9) [reply modules/ROOT/pages/afm-create-monitor.adoc:71]: Good point. Though in this case, it's referring to prominent UI table rows. Each row represents a day, so I'm thinking it might be appropriate to say that they are "separated by day".

File: modules/ROOT/pages/afm-create-private-location.adoc [hunk 28]
```diff
@@ -7,29 +7,28 @@ endif::[]
...
 
-* When you create a monitor
-* When you add a schedule to a monitor
-* When you modify a schedule in a monitor
+* When you xref:afm-create-monitor.adoc[create a monitor]
+* When you xref:afm-add-schedule.adoc[add a schedule] to a monitor
+* When you xref:afm-modify-schedule.adoc[modify a schedule] in a monitor
 
-You can also create a private location independently of these three tasks. Then, when you create a monitor, add a schedule, or modify a schedule, you can simply select the private location. The following procedure explains the steps for this last method.
+When you create a monitor, add a schedule, or modify a schedule, you can select the private location. This procedure explains the steps for this last method.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-private-location.adoc]: The first sentence here seems to just repeat the list intro and list.
How about something like...:

<what a private location is and under what circumstances your audience wants or needs to create one>
"You have the opportunity to create a private location when you:

- Create...
- Add...
- Modify...

This procedure explains how to create a private location when you modify...."

File: modules/ROOT/pages/afm-create-private-location.adoc [hunk 29]
```diff
@@ -7,29 +7,28 @@ endif::[]
...
 
-For information about what a private location is and why you might want to create one, see xref:afm-monitoring-private-apis.adoc[Monitoring the Endpoints of Private APIs].
+For information about private locations and their uses, see xref:afm-monitoring-private-apis.adoc[Monitoring the Endpoints of Private APIs].
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-private-location.adoc:16]: So, I think one or two sentences about this should begin the topic.

File: modules/ROOT/pages/afm-create-private-location.adoc [hunk 30]
```diff
@@ -7,29 +7,28 @@ endif::[]
...
 
 == Before You Begin
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-private-location.adoc:18]: We will establish a template for Tasks soon, and it will have boilerplate headings like this, Procedure, Results, and so on. I hope. Maybe. I dunno', I expect a lot of pushback....

File: modules/ROOT/pages/afm-create-private-location.adoc [hunk 31]
```diff
@@ -7,29 +7,28 @@ endif::[]
...
 
-Ensure that your organization in Anypoint Platform has a subscription to Anypoint Virtual Private Cloud.
+Ensure that your Anypoint Platform organization has a subscription to Anypoint Virtual Private Cloud.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-private-location.adoc:20]: Anypoint Platform Private Cloud Edition is the only product name that I can find that comes close to this.
There is no "Anypoint Virtual Private Cloud" as a named entity, as far as I can tell.

File: modules/ROOT/pages/afm-create-private-location.adoc [hunk 32]
```diff
@@ -7,29 +7,28 @@ endif::[]
...
 
 == Procedure
 
-To create a private location independently of any other task:
+To create a private location:
 
-. Display the list of locations by clicking the *Locations* tab at the top of the screen.
-. Click *Create Private Location*.
+. When creating a test or schedule, tap the field under *Select location* and choose *Create new location*.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-private-location.adoc]: GLOBALLY replace "tap" and "choose" with the verbs we use.

Comment (kbostrom) [reply modules/ROOT/pages/afm-create-private-location.adoc]: Also, avoid using "under" unless something is visibly beneath something else in the UI. 
I typically see "in": "In *Select location*, select *Create new location*."

File: modules/ROOT/pages/afm-create-private-location.adoc [hunk 33]
```diff
@@ -7,29 +7,28 @@ endif::[]
...
 . Name the location.
 +
 The name can be up to 255 characters, and can include spaces and special characters.
-. Select the environment in which to run the workers that are created to run tests in your monitors.
-. Click *Create*.
+. Select the environment where the tests will run.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-private-location.adoc:30]:
```suggestion
. Select a test environment.
```

File: modules/ROOT/pages/afm-create-private-location.adoc [hunk 34]
```diff
@@ -7,29 +7,28 @@ endif::[]
...
+. Complete the creation of the of test or schedule. The new location is included.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-create-private-location.adoc]:
```suggestion
. Finish creating the test or schedule.
```
Isn't "The new location is included." part of the result?
It's definitely not part of the step of completing the process.

File: modules/ROOT/pages/afm-delete-monitor.adoc [hunk 35]
```diff
@@ -5,14 +5,25 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can delete a monitor that you no longer need. However, there is no cost to keeping the monitor if it is not scheduled to run.
+Unneeded monitors can be easily deleted. However, there is no cost in keeping monitors if not scheduled to run.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-delete-monitor.adoc]: Reword to active voice:
```suggestion
Although you can easily delete unnecessary monitors, there is no cost in keeping them.
```

File: modules/ROOT/pages/afm-delete-monitor.adoc [hunk 36]
```diff
@@ -5,14 +5,25 @@ endif::[]
...
 
 == Procedure
 
+There are two methods to delete a monitor.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-delete-monitor.adoc]: Avoid beginning sentences with "There is" or "There are":
```suggestion
Delete a monitor either from the homepage or from its detail page.
```

File: modules/ROOT/pages/afm-delete-monitor.adoc [hunk 37]
```diff
@@ -5,14 +5,25 @@ endif::[]
...
+. Click *Delete* (image:afm-ui-trash-button.png[width=1.5%,height=1.5%]).
+. In the *Delete Monitor* dialog, click *Delete*.
+
+To delete a monitor from its detail page:
+
 . Select the monitor that you want to delete.
-. In the top-right corner of the screen, click the *Delete* button that is next to the trash-can icon.
-. In the Delete Monitor dialog, click *Delete*.
+. Click the monitor.
+. Tap *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) followed by *Delete Monitor*.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-delete-monitor.adoc]: Ditto prev. suggestion to reword:
```suggestion
. Click *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) > *Delete Monitor*.
```

**Review by kbostrom — DISMISSED** [review]
CE create private location and delete monitor .adocs.
Complete PR edit done.

File: modules/ROOT/pages/afm-create-private-location.adoc (hunk 32 above)
Comment (chris-campbell9) [reply modules/ROOT/pages/afm-create-private-location.adoc]: Yeah, it's beneath the **Select Location** UI label.

File: modules/ROOT/pages/afm-create-private-location.adoc (hunk 33 above)
Comment (chris-campbell9) [reply modules/ROOT/pages/afm-create-private-location.adoc:30]: Hmm, this is an interesting point. I'm thinking it should remain as is because we're not talking about a test environment (this means something else entirely).

File: modules/ROOT/pages/afm-create-private-location.adoc (hunk 31 above)
Comment (chris-campbell9) [reply modules/ROOT/pages/afm-create-private-location.adoc:20]: I think this is referring to Anypoint Virtual Private Cloud (VPC):
https://docs.mulesoft.com/runtime-manager/virtual-private-cloud

File: modules/ROOT/pages/afm-add-schedule.adoc (hunk 6 above)
Comment (chris-campbell9) [reply modules/ROOT/pages/afm-add-schedule.adoc]: These values are actually list options in the UI. As far as I can tell, they're always in 12-hour time.

File: modules/ROOT/pages/afm-add-schedule.adoc (hunk 4 above)
Comment (chris-campbell9) [reply modules/ROOT/pages/afm-add-schedule.adoc]: Ok cool, I reworded for clarity. Hopefully this is better:
"The minute at which the test begins, starting 60 minutes after you create the schedule"

I don't think we need to include an example since the "minutes" value is a list option in the UI.

File: modules/ROOT/pages/afm-create-private-location.adoc (hunk 29 above)
Comment (chris-campbell9) [reply modules/ROOT/pages/afm-create-private-location.adoc:16]: Cool, I added an intro sentence that defines "private location". There's an existing link to its uses, so I'm thinking it would be superfluous to mention any additional info.

File: modules/ROOT/pages/afm-create-monitor.adoc (hunk 16 above)
Comment (chris-campbell9) [reply modules/ROOT/pages/afm-create-monitor.adoc:20]: I addressed the lack of clarity elsewhere, so I think we're good.

**Review by kbostrom — APPROVED** [review]
RA
